plugins/mana/plugin.go

```diff
@@ -281,27 +285,42 @@ func shutdownStorages() {
 // It also updates the mana values for each node.
 // If n is zero, it returns all nodes.
 func GetHighestManaNodes(manaType mana.Type, n uint) ([]mana.Node, time.Time, error) {
+	if !QueryAllowed() {
+		return []mana.Node{}, time.Now(), ErrQueryNotAllowed
```

Comment:
Should we use `clock.SyncedTime()` rather than time.Now()? 
As a side note, should we do the same for whenever we use time within Mana?

Reply:
I don't think so. The time here is the node's own time.

Reply:
we could

Reply:
> I don't think so. The time here is the node's own time.

but the node's own time should be `clock.SyncedTime()`

Reply:
anyhow, I guess that with this PR is ok since the content is empty. Then, we re-evaluate the time usage in the rest of mana